ui/src/pages/publications/[id]/versions/[versionId].tsx

```diff
@@ -215,6 +215,7 @@ const Publication: Types.NextPage<Props> = (props): React.ReactElement => {
 
     const list = [];
 
+    const showAdditionalInformation = Boolean(publicationVersion?.additionalInformation.length);
```

Comment:
Curious way to cast to a boolean (I know you're keeping things standard with what's already there).. common way I use is `!!publicationVersion?.additionalInformation.length`

Reply:
It does read a bit old fashioned. I'd probably use `!!` too in most cases. I do like how explicit this way is at least, even if it is longer